cmd/kops/validate_cluster.go

```diff
@@ -172,8 +172,8 @@ func RunValidateCluster(f *util.Factory, cmd *cobra.Command, args []string, out
 		// do we need to print which instance group is not ready?
 		// nodes are going to be a pain
 		fmt.Fprint(out, "\nValidation Failed\n")
-		fmt.Fprintf(out, "Master(s) Not Ready %d out of %d.\n", len(validationCluster.MastersNotReadyArray), validationCluster.MastersCount)
-		fmt.Fprintf(out, "Node(s) Not Ready   %d out of %d.\n", len(validationCluster.NodesNotReadyArray), validationCluster.NodesCount)
+		fmt.Fprintf(out, "Ready Master(s) %d out of %d.\n", len(validationCluster.MastersNotReadyArray), validationCluster.MastersCount)
+		fmt.Fprintf(out, "Ready Node(s) %d out of %d.\n", len(validationCluster.NodesNotReadyArray), validationCluster.NodesCount)
```

Comment:
This way the message will be `Ready node(s) 2 out of 5` which indicates that 2 nodes are up and 3 are not. I can change the message if it is not clear enough.